### ISSUE TITLE
set up mailto with default subject line

### ISSUE THREAD

**Issue by bridgetkromhout** [devopsdays/devopsdays-web#262]
Is there a hugo equiv of this?
```
[drop us an email](mailto:<%= render(:partial => "/#{@eventhome}/_email_organizers") %>?subject=Sponsor devopsdays Washington, DC 2016).
```

If not, we might want to make one. DC, among others, was using it.

**Comment by mattstratton** [comment]
Yep. Jerry was asking for this too. I plan to extend the functionality of the mailing shortcodes.

**Comment by mattstratton** [comment]
This was fixed in #900
